streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/ThreadMetricsTest.java

```diff
@@ -75,7 +78,6 @@
     @Before
     public void setUp() {
         expect(streamsMetrics.version()).andReturn(builtInMetricsVersion).anyTimes();
-        mockStatic(StreamsMetricsImpl.class);
```

Comment:
I moved this into the tests where it was needed because it was interfering with the new tests, in which we specifically need the static method _not_ to be mocked so that we can make sure that we are actually adding the desired metrics. The tests with this thing mocked only verify that ThreadMetrics invokes the static method on StreamsMetricsImpl, not that it actually creates the right metrics, but I don't have the ambition to fix the rest of the tests right now.